### ISSUE TITLE
flaky tabbing order test

### ISSUE THREAD

**Issue by Xymmer** [SAP/spartacus#9153]
Decoupling the test into chunks, where we will put each accessibility test to each corresponding feature.

**All scenarios should be taken from the use case in `projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.flaky-e2e-spec.ts`** 
1)  make sure the files are ending with `.e2e-spec.ts` as some e2es are quite old
2) make sure when bringing over the accessbility test case to the corresponding feature e2e test, you will make the accessibility part the **last sequence of the e2e test.**
3) When making it the last sequence, **encapsulate the test case under a describe('Accessibility')** that covers both test cases of being logged in or not IF it applies

- [ ] `NEW` e2e for testing the _header, my-account, and footer_ tabbing parts. `navigation.e2e-spec.ts`
- [ ] adding the _homepage_ tabbing parts to `homepage.e2e-spec.ts`
- [ ] adding the _login page_ tabbing parts to `login.e2e-spec.ts` 
- [ ] adding the _register page_ tabbing parts to `register.e2e-spec.ts`
- [ ] adding the _reset page_ tabbing parts to `reset-password.e2e-spec.ts`
- [ ] adding the _product list_ tabbing parts to `product-search.e2e-spec.ts`
- [ ] adding the _product page and product page tabs_ tabbing parts to `product-details.e2e-spec.ts`
- [ ] adding the _Add to cart_ tabbing parts to `added-to-cart-modal.e2e-spec.ts`
- [ ] adding the _cart_ tabbing parts to `cart.e2e-spec.ts`
- [ ] adding the _store finder_ tabbing parts to `store-finder.e2e-spec.ts`
- [ ] adding the _stock notification_ tabbing parts to `stock-notification.e2e-spec.ts`
- [ ] adding the _asm_ tabbing parts to `asm.e2e-spec.ts`
- [ ] adding the _general checkout and generic order details_ tabbing parts to `checkout-flow.e2e-spec.ts`
- [ ] adding the _order history no orders_ tabbing parts to `order-history-no-orders-flow.e2e-spec.ts`
- [ ] adding the _order history WITH orders_ tabbing parts to `order-history-orders-flow.e2e-spec.ts`
- [ ] adding the _notification preference_ tabbing parts to `notification-preference.e2e-spec.ts`
- [ ] adding the _change password_ tabbing parts to `update-password.e2e-spec.ts`
- [ ] adding the _personal details_ tabbing parts to `update-profile.e2e-spec.ts`
- [ ] adding the _update email_ tabbing parts to `update-email.e2e-spec.ts`
- [ ] adding the _close account_ tabbing parts to `close-account.e2e-spec.ts`
- [ ] adding the _consent management_ tabbing parts to `consent-management.e2e-spec.ts`
- [ ] adding the _address book (form)_ tabbing parts to `address-book.e2e-spec.ts`
- [ ] adding the _payment details_ tabbing parts to `payment-methods.e2e-spec.ts`
- [ ] adding the _wish list_ tabbing parts to `wish-list.e2e-spec.ts`
- [ ] adding the _save for later_ tabbing parts to `save-for-later.e2e-spec.ts`
- [ ] adding the _my interest_ tabbing parts to `my-interests.e2e-spec.ts`
- [ ] adding the _coupons_ tabbing parts to `coupon.e2e-spec.ts`
- [ ] adding the _my-coupons_ tabbing parts to `my-coupons.e2e-spec.ts`
- [ ] adding the _consignment_  tabbing parts to `consignment-tracking.e2e-spec.ts`

**Comment by Zeyber** [comment]
We can probably close this in favour of #13686 and subsequent a11y configurations once they are merged.
